docs/create-packages/creating-a-package-dotnet-cli.md

```diff
@@ -0,0 +1,147 @@
+
+For .NET Core and .NET Standard projects that use the [SDK-style format](../resources/check-project-format.md), and any other SDK-style projects, NuGet uses information in the project file directly to create a package. For detailed steps, see [Create .NET Standard Packages with dotnet CLI](../quickstart/create-and-publish-a-package-using-the-dotnet-cli.md), [Create .NET Standard Packages with Visual Studio](../quickstart/create-and-publish-a-package-using-visual-studio.md) or [NuGet pack and restore as MSBuild targets](../reference/msbuild-targets.md).
+
+Technically speaking, a NuGet package is just a ZIP file that's been renamed with the `.nupkg` extension and whose contents match certain conventions. This topic describes the detailed process of creating a package that meets those conventions.
+
+> [!Important]
+> This topic applies to [SDK-style](../resources/check-project-format.md) projects, typically .NET Core and .NET Standard projects.
+
+## The role and structure of the .nuspec file
+
+A `.nuspec` file is not required to create packages for SDK-style projects (typically .NET Core and .NET Standard projects that use the [SDK attribute](/dotnet/core/tools/csproj#additions)). However, a `.nuspec` is generated when you create the package.
+
+If you are creating a package using `dotnet.exe pack` or `msbuild pack target`, we recommend that you [include all the properties](../reference/msbuild-targets.md#pack-target) that are usually in the `.nuspec` file in the project file instead. However, you can instead choose to [use a `.nuspec` file to pack using `dotnet.exe` or `msbuild pack target`](../reference/msbuild-targets.md#packing-using-a-nuspec).
+
+## Set properties
+
+The following properties are required to create a package.
+
+- `PackageId`, the package identifier, which must be unique across the gallery that hosts the package.
+- `Version`, a specific version number in the form *Major.Minor.Patch[-Suffix]* where *-Suffix* identifies [pre-release versions](prerelease-packages.md)
+- The package title as it should appear on the host (like nuget.org)
+- `Authors`, author and owner information.
+- `Company`, your company name.
+
+In Visual Studio, you can set these values in the project properties (right-click the project in Solution Explorer, choose **Properties**, and select the **Package** tab). You can also set these properties directly in the project files (`.csproj`).
+
+    ```xml
+    <PropertyGroup>
+      ...
+      <PackageId>AppLogger</PackageId>
+      <Version>1.0.0</Version>
+      <Authors>your_name</Authors>
+      <Company>your_company</Company>
+    </PropertyGroup>
+    ```
+
+> [!Important]
+> Give the package an identifier that's unique across nuget.org or whatever host you're using.
```

Comment:
```suggestion
> Give the package an identifier that's unique across nuget.org or whatever package source you're using.
```

just a suggestion, but "package source" is more inline with the NuGet's vocabulary, but it might not be clear to people new to the ecosystem.